numpy/ma/extras.py

```diff
@@ -48,7 +48,7 @@
 from numpy import ndarray, array as nxarray
 import numpy.core.umath as umath
 from numpy.lib.index_tricks import AxisConcatenator
-from numpy.linalg import lstsq
+from numpy.lib.function_base import _CORRCOEF_MSG_FMT, _DefaultArg
```

Comment:
Just repeat the definition here. This import complicates the code, obscures the value of _CORRCOEF_MSG_FMT, and introduces a dependency between modules. The format is temporary anyway. No reason not to also put the private `_DefaultArg` here also, and for the same reasons.

_DefaultArg probably belongs somewhere else where it can be reused.